PurchasesTests/Networking/HTTPClientTests.swift

```diff
@@ -803,19 +803,26 @@ private extension HTTPRequest.Path {
 
 }
 
-private extension HTTPRequest.Method {
+extension HTTPRequest.Method {
 
-    static func requestNumber(_ number: Int) -> Self {
-        return .post(body: [HTTPClientTests.requestNumberKeyName: number])
+    fileprivate static func requestNumber(_ number: Int) -> Self {
+        return .post([HTTPClientTests.requestNumberKeyName: number])
     }
 
-    static func invalidBody() -> Self {
+    fileprivate static func invalidBody() -> Self {
```

Comment:
unrelated: I kinda feel like we should at one point move all `private`s to `fileprivate`, I don't think we ever _intend_ to use `private`

Reply:
Having `private` in the extension makes the methods `fileprivate`. But since I needed one of these to be `internal`, I had to make that explicit.

Reply:
> Having private in the extension makes the methods fileprivate.

TIL